sections/predictive-search.liquid

```diff
@@ -1,71 +1,149 @@
     {%- endif -%}
-    <ul
-      id="predictive-search-results-list"
-      class="predictive-search__results-list list-unstyled"
-      role="listbox"
-      aria-labelledby="predictive-search-products"
-    >
-      {%- for product in predictive_search.resources.products -%}
-        <li
-          id="predictive-search-option-{{ forloop.index }}"
-          class="predictive-search__list-item"
-          role="option"
-          aria-selected="false"
-        >
-          <a
-            href="{{ product.url }}"
-            class="predictive-search__item predictive-search__item--link link link--text"
-            tabindex="-1"
-          >
-            {%- if product.featured_media != blank -%}
-              <img
-                class="predictive-search__image"
-                src="{{ product.featured_media | image_url: width: 150 }}"
-                alt="{{ product.featured_media.alt }}"
-                width="50"
-                height="{{ 50 | divided_by: product.featured_media.preview_image.aspect_ratio }}"
-              >
-            {%- endif -%}
-            <div class="predictive-search__item-content{% unless settings.predictive_search_show_vendor or settings.predictive_search_show_price %} predictive-search__item-content--centered{% endunless %}">
-              {%- if settings.predictive_search_show_vendor -%}
-                <span class="visually-hidden">{{ 'accessibility.vendor' | t }}</span>
-                <div class="predictive-search__item-vendor caption-with-letter-spacing">{{ product.vendor }}</div>
-              {%- endif -%}
-              <h3 class="predictive-search__item-heading h5">{{ product.title }}</h3>
-              {%- if settings.predictive_search_show_price -%}
-                {% render 'price', product: product, use_variant: true, show_badges: false %}
-              {%- endif -%}
+      {%- if first_column_results_size > 0 -%}
+        <div class="predictive-search__result-group">
+      {%- endif -%}
+      {%- if predictive_search.resources.queries.size > 0 or predictive_search.resources.collections.size > 0 -%}
+        <div>
+          <h2 id="predictive-search-queries" class="predictive-search__heading text-body caption-with-letter-spacing">
+            {{ 'templates.search.suggestions' | t }}
+            <svg aria-hidden="true" focusable="false" role="presentation" class="spinner" viewBox="0 0 66 66" xmlns="http://www.w3.org/2000/svg">
+              <circle class="path" fill="none" stroke-width="6" cx="33" cy="33" r="30"></circle>
+            </svg>
+          </h2>
+          <ul id="predictive-search-results-queries-list" class="predictive-search__results-list list-unstyled" role="group" aria-labelledby="predictive-search-queries">
+            {%- for query in predictive_search.resources.queries -%}
+              <li id="predictive-search-option-query-{{ forloop.index }}" class="predictive-search__list-item" role="option" aria-selected="false">
```

Comment:
```suggestion
              <li id="predictive-search-option-query-{{ forloop.index }}" class="predictive-search__list-item" role="option" aria-selected="false" aria-label="{{ query.text }}">
```

I was testing in Safari + VoiceOver and Mobile Safari + VoiceOver and realized that it was interpreting the styled text as two separate enteries. It would read `p<mark>rime</mark>` as "P Rime" instead of "Prime".

Reply:
Good point 👍🏻